### PR TITLE
[SDPA-5327] Add User help links in the CMS

### PR DESCRIPTION
**Pull request by mayurngondhkar**
Jira: https://digital-engagement.atlassian.net/browse/SDPA-5327

Changes:

1) Create a new module "tide_cms_support". This supports the addition of future support related content.

### CODE REVIEW

File: modules/tide_cms_support/tide_cms_support.links.menu.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      target: _blank
+tide_cms_support.cms_content_templates:
+  title: 'Content templates'
+  description: 'How to use our content page templates or create specific types of content.'
+  parent: 'tide_cms_support.cms_guides'
+  url: https://www.singledigitalpresence.vic.gov.au/content-types-and-templates
+  weight: -40
+  options:
+    attributes:
+      target: _blank
```
Comment (MdNadimHossain) [inline modules/tide_cms_support/tide_cms_support.links.menu.yml:27]: :ok_hand:

File: tide_core.install [hunk 2]
```diff
@@ -1331,3 +1331,20 @@ function tide_core_update_8054() {
...
+/**
+ * Check if the "tide_cms_support" module is installed
+ */
+function tide_core_update_8055() {
+  $module_handler = \Drupal::module_handler();
+  $module_exist = $module_handler->moduleExists('tide_cms_support');
+
+  $modules = [
+    'tide_cms_support'
+  ];
```
Comment (MdNadimHossain) [inline tide_core.install:1344]: do not need this array, as only one module is getting installed

File: tide_core.install [hunk 3]
```diff
@@ -1331,3 +1331,20 @@ function tide_core_update_8054() {
...
+
+  if (!$moduleExists) {
+    $module_installer = \Drupal::service('module_installer');
+    $module_installer->install($modules);
```
Comment (MdNadimHossain) [inline tide_core.install:1348]: you can do it like this - `$module_installer->install(['tide_cms_support']);`

File: tide_core.install [hunk 4]
```diff
@@ -1331,3 +1331,20 @@ function tide_core_update_8054() {
     $config->set('cardinality', -1)->save();
   }
 }
+
+/**
+ * Check if the "tide_cms_support" module is installed
```
Comment (MdNadimHossain) [inline tide_core.install:1336]: Update the comment to say - Enable "tide_cms_support" module, if it is not enabled.

**Review by MdNadimHossain — CHANGES_REQUESTED** [review]
@mayurngondhkar minor changes needed and behat test needs to be included.

File: tide_core.install (hunk 3 above)
Comment (mayurngondhkar) [reply tide_core.install:1348]: Resolved

File: tide_core.install (hunk 4 above)
Comment (mayurngondhkar) [reply tide_core.install:1336]: Resolved

File: tide_core.install (hunk 2 above)
Comment (mayurngondhkar) [reply tide_core.install:1344]: Resolved

**Comment by mayurngondhkar** [comment]
NOTE: REVERT [c5605cc](https://github.com/dpc-sdp/tide_core/pull/293/commits/c5605cc0374ae336b3d14c2971a43298ed696933) once it's approved in DEV TOOLS

**Comment by mayurngondhkar** [comment]
NOTE: Revert [c5605cc](https://github.com/dpc-sdp/tide_core/pull/293/commits/c5605cc0374ae336b3d14c2971a43298ed696933) once approved in dev tools

File: composer.json [hunk 5]
```diff
@@ -224,7 +224,7 @@
                 "More defensive handling of data-entity-embed-display-settings - https://www.drupal.org/project/entity_embed/issues/3010942": "https://www.drupal.org/files/issues/2019-12-11/3077225-10.reduce-invalid-config-logs.patch"
             },
             "drupal/metatag": {
-                "Support json api - https://www.drupal.org/project/metatag/issues/2945817": "https://www.drupal.org/files/issues/2021-12-21/metatag-data-type-support-2945817-132.patch"
+                "Support json api - https://www.drupal.org/project/metatag/issues/2945817": "https://www.drupal.org/files/issues/2022-02-21/metatag-n2945817-143.patch"
```
Comment (MdNadimHossain) [inline composer.json:227]: cross check this patch with @GROwen as this is one of the patch created by him.

File: dev-tools.sh [hunk 6]
```diff
@@ -33,6 +33,6 @@
 # stable version.
 #
 # Uncomment and set the Dev-Tools's commit value and commit this change.
-# export GH_COMMIT=COMMIT_SHA
+export GH_COMMIT=7f0ad2e879dddeda3d9113c9fcfde134b2a775ac
```
Comment (MdNadimHossain) [inline dev-tools.sh:36]: remember to remove this one before merging this PR and after the dev-tools is released with your changes

**Review by MdNadimHossain — APPROVED** [review]
Everything else looks good to me, just follow up with the comments before merging

File: dev-tools.sh (hunk 6 above)
Comment (mayurngondhkar) [reply dev-tools.sh]: This has been removed.

**Comment by mayurngondhkar** [comment]
The build fail issue has been fixed in dev-tools

**Comment by mayurngondhkar** [comment]
Build Fail as changes are dependent on dev-tools. Dev tools change merged. Read comments above.

**Pull request merged**